src/editing_tools/AdjustTool.vala

```diff
@@ -216,7 +214,7 @@ public class EditingTools.AdjustTool : EditingTool {
         }
 
         public override bool compress (Command command) {
-            AdjustResetCommand reset_command = command as AdjustResetCommand;
+            var reset_command = (AdjustResetCommand) command;
             if (reset_command == null) {
                 return false;
             }
```

Comment:
Oh wait, if we change the way of casting from `as AdjustResetCommand` to `(AdjustResetCommand)`, the `if` sentence following from the next lines might no longer work? Casting by `as` can return null but the latter don't and maybe throws an error I guess, as I read [the wiki](https://wiki.gnome.org/Projects/Vala/Tutorial#:~:text=gobject%2Dstyle%20construction.-,Dynamic%20Type%20Casting,-For%20the%20dynamic) (sorry if I'm misreading).

Reply:
I read about it too on the code style, it says it throws a null. but since the code style says we should avoid `as`, I guess we can just remove the if statement which follows.